### PR TITLE
Update SVNManager.java

### PR DESCRIPTION
**Pull request by jaadams5**
Change error message as requested.

### CODE REVIEW

File: src/net/sourceforge/kolmafia/svn/SVNManager.java [hunk 1]
```diff
@@ -515,8 +515,7 @@ private static boolean validateRepo(SVNURL repo, boolean quiet) {
```
Comment (heeheehee-kolmafia) [inline src/net/sourceforge/kolmafia/svn/SVNManager.java:515]: hm, it looks like failed is potentially being cleared depending on loop order, right before this (in the dependencies.txt check; should probably read failed |= ...).

(I was curious whether we could provide a better error message for _why_ it failed validation, in addition to what as you have here.)

**Review by heeheehee-kolmafia — APPROVED** [review]

File: src/net/sourceforge/kolmafia/svn/SVNManager.java (hunk 1 above)
Comment (jaadams5) [reply src/net/sourceforge/kolmafia/svn/SVNManager.java:515]: I couldn't really get a useful debugger run without actually changing a repository and that was more work than just making the change.   I wondered about a better message but I wasn't sure I wanted to understand the SVN code well enough to actually make an intentional change.  Worth revisiting if it is not made obsolete by adding git support to KoLmafia or refactoring how scripts get installed and updated.

File: src/net/sourceforge/kolmafia/svn/SVNManager.java [hunk 2]
```diff
@@ -515,8 +515,7 @@ private static boolean validateRepo(SVNURL repo, boolean quiet) {
 
     if (failed && !quiet) {
       KoLmafia.updateDisplay(
-          MafiaState.ERROR,
-          "The requested repo failed validation.  Complain to the script's author.");
+          MafiaState.ERROR, "The requested repo (" + repo.getPath() + " failed validation.");
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/svn/SVNManager.java:518]: @jaadams5  This is missing a closing bracket

Comment (jaadams5) [reply src/net/sourceforge/kolmafia/svn/SVNManager.java:518]: @gausie 
Would you recheck, please?  On my system the only diff is in the string being passed to updateDisplay.

If failed has a an opening bracket, the else has the closing and then an opening and then there is a close before the return.

Comment (BadHorseMonkey) [reply src/net/sourceforge/kolmafia/svn/SVNManager.java:518]: I think he means in the error text.

+          MafiaState.ERROR, "The requested repo (" + repo.getPath() + "**)** failed validation.");

Comment (jaadams5) [reply src/net/sourceforge/kolmafia/svn/SVNManager.java:518]: See https://github.com/kolmafia/kolmafia/pull/73

I tend to use "bracket" for { or [ and "parenthesis" for ( so I was looking for the wrong thing.

**Pull request merged**